gammapy/analysis/config.py

```diff
@@ -138,6 +138,12 @@ class FitConfig(GammapyBaseConfig):
     fit_range: EnergyRangeConfig = EnergyRangeConfig()
 
 
+class ExcessMapConfig(GammapyBaseConfig):
+    correlation_radius: AngleType = '0.1 deg'
+    parameters: dict = {"selection_optional": ""}
```

Comment:
I think `"selection_optional": ""` can be safely removed, there are other possible parameters allowed, so better declare only those that we want to set by default. Let's make `parameters: dict = {}` like in the other classes.

Reply:
Indeed. The default is now `selection_optional=None` so we don't need to pass anything.